src/test/java/edu/harvard/iq/dataverse/DatasetFieldServiceBeanDansTest.java

```diff
@@ -0,0 +1,103 @@
+package edu.harvard.iq.dataverse;
+
+import edu.harvard.iq.dataverse.settings.SettingsServiceBean;
+import jakarta.json.Json;
+import jakarta.json.JsonObject;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.AdditionalMatchers;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class DatasetFieldServiceBeanDansTest {
+
+    private DatasetFieldServiceBean datasetFieldServiceBean;
+
+    static String getCvocJson(String pathToJsonFile) throws IOException {
+        final File datasetVersionJson = new File(pathToJsonFile);
+        return new String(Files.readAllBytes(Paths.get(datasetVersionJson.getAbsolutePath())));
+    }
+
+    @BeforeEach
+    void setUp() {
+      this.datasetFieldServiceBean = Mockito.spy(new DatasetFieldServiceBean());
+    }
+
+    @AfterEach
+    void tearDown() {
+      this.datasetFieldServiceBean = null;
+    }
+
+    @Test
+    void getIndexableStringsForAudience() throws Exception {
+        String termURI = "https://www.narcis.nl/classification/D13000";
+        JsonObject cvocEntry = createMocks("dansAudience", termURI, "audience.json");
+        JsonObject readObject = readObject("src/test/resources/json/cvoc-dans-value/audience.json");
+
+        String result = reflectFilterResponse().invoke(datasetFieldServiceBean, cvocEntry, readObject, termURI).toString();
+
+        assertTrue(result.contains(termURI));
+        assertTrue(result.contains("Exploitation and management of the physical environment"));
+        assertTrue(result.contains("Exploitatie en beheer van het fysieke milieu"));
+    }
+
+    @Test
+    void getIndexableStringsForAbrPeriod() throws Exception {
```

Comment:
Expected this one to be OK as the VM doesn't produce the logging that is produced by this test.